app/platform/fabric/gateway/FabricGateway.ts

```diff
@@ -47,6 +48,7 @@ export class FabricGateway {
 		this.gateway = new Gateway();
 		this.fabricCaEnabled = false;
 		this.client = null;
+		this.clientTlsIdentity = false;
```

Comment:
It's better initialising with null rather than false. And if possible, can you define this member with an exact type (Identity)?
https://github.com/hyperledger/fabric-sdk-node/blob/master/fabric-network/src/impl/wallet/wallet.ts#L50

Reply:
I fully agree with the comment, and amended them

Reply:
It seems to fail transcode of typescript. Please refer the following code for fixing transcode error.
https://github.com/hyperledger/fabric-sdk-node/blob/master/fabric-network/src/gateway.ts#L312

Reply:
I see...  now test failed

I am going to make corrections in relation to it, and I think we can further refine it structurally.
but currently impossible to edit due to personal time, so I will commit again in 24 hours.

Reply:
@nekia 
Done, waiting for further comments

Reply:
It looks good to me.